core/src/main/java/com/netflix/iceberg/BaseTableScan.java

```diff
@@ -223,12 +262,16 @@ public Expression filter() {
     return rowFilter;
   }
 
+  @Override
+  public boolean isCaseSensitive() { return caseSensitive; }
```

Comment:
Nit: please use expanded formatting and collapse in your editor.

Reply:
Ack.